fmriprep/cli/run.py

```diff
@@ -5,7 +5,6 @@
 fMRI preprocessing workflow
 =====
 """
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
I would discontinue py2 for good at this point, simplify our lives. But you are right, we haven't talked about this. WDYT @chrisfilo @rwblair ?